docs/content/3.guide/1.writing/7.document-driven.md

```diff
@@ -111,7 +111,7 @@ This option will search for a `theme` key in `globals`, then search for a `layou
 
 ## Global variables
 
-Queries are be made from a [route middlewares](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
+Queries are been made from a [route middlewares](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
```

Comment:
```suggestion
Queries are made from a [route middleware](https://v3.nuxtjs.org/guide/directory-structure/middleware#middleware-directory) and are resolved before your page renders.
```

Reply:
much better. Thanks... I made a mess of correction. It should be `being` not `been`.